src/FluentMigrator.Runner.Postgres/Generators/Postgres10/Postgres10Column.cs

```diff
@@ -0,0 +1,91 @@
+#region License
+//
+// Copyright (c) 2019, Fluent Migrator Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#endregion
+
+using FluentMigrator.Infrastructure.Extensions;
+using FluentMigrator.Model;
+using FluentMigrator.Postgres;
+using FluentMigrator.Runner.Generators.Postgres;
+
+using JetBrains.Annotations;
+
+using System;
+
+namespace FluentMigrator.Runner.Generators.Postgres10
+{
+    internal class Postgres10Column : PostgresColumn
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Postgres10Column"/> class.
+        /// </summary>
+        /// <param name="quoter">The Postgres quoter.</param>
+        /// <param name="typeMap">The Postgres type map.</param>
+        public Postgres10Column([NotNull] PostgresQuoter quoter, ITypeMap typeMap)
+            : base(quoter, typeMap)
+        {
+        }
```

Comment:
In my branch I added to this file, a private `FormatAlterIdentity` method, which I added to the the base 
class's `AlterClauseOrder` protected list here in the constructor, so I could make use of the base class implementation of `GenerateAlterClauses` without override, to achieve the new extension methods to Add/Set/Drop/Drop If Exists identity generation on existing columns.